servers/rendering/rendering_device_driver.h

```diff
@@ -301,7 +302,7 @@ class RenderingDeviceDriver : public RenderingDeviceCommons {
 	/**** VERTEX ARRAY ****/
 	/**********************/
 
-	virtual VertexFormatID vertex_format_create(VectorView<VertexAttribute> p_vertex_attribs) = 0;
+	virtual VertexFormatID vertex_format_create(Span<VertexAttribute> p_vertex_attribs, const VertexAttributeBindingsMap &p_vertex_bindings) = 0;
```

Comment:
Switching to Span means we can avoid allocations at call sites. We should evaluate all calls and use FixedVector, as the sizes in the renderer_rd are all known at compile time.